sw/ground_segment/python/gvf/gvfFormation.py

```diff
@@ -111,9 +111,9 @@ def main():
         interface.shutdown()
         return
 
-    B = np.loadtxt(sys.argv[1])
-    desired_sigmas = np.loadtxt(sys.argv[2])
-    ids = np.loadtxt(sys.argv[3])
+    B = np.loadtxt('./formation/' + sys.argv[1])
```

Comment:
not sure it is interesting to force a local path to `formation` folder, you may want to have your input files in different directories or even pass an absolute path